game_event/tests.py

```diff
@@ -31,6 +34,34 @@ def saved_game_event_player(saved_game_event, player):
                                           ball_responsible=False)
 
 
+@pytest.fixture
+def court_ball_game(court):
+    return CourtBallGame.objects.create(court=court, ball_game=TEST_BALL_GAME)
+
+
+@pytest.fixture
+def mock_urlopen():
+    with mock.patch('urllib.request.urlopen') as mock_urlopen:
+        return mock_urlopen
+
+
+@pytest.fixture
+def saved_game_events_list(saved_game_event, court):
+    game_events_list = []
+    game_event = GameEvent.objects.create(
+        id=1001,
+        time=timezone.now() - timezone.timedelta(days=1),
+        level_of_game=5,
+        min_number_of_players=4,
+        max_number_of_players=10,
+        court=court,
+        ball_game='Basketball'
+    )
+    game_events_list.append(game_event)
+    game_events_list.append(saved_game_event)
+    return game_events_list
```

Comment:
It's more readable if you visually put the objects inside a list rather than using append.
```suggestion
    game_event = GameEvent.objects.create(
        id=1001,
        time=timezone.now() - timezone.timedelta(days=1),
        level_of_game=5,
        min_number_of_players=4,
        max_number_of_players=10,
        court=court,
        ball_game='Basketball'
    )
    return [
        game_event,
        saved_game_event,
    ]
```

Reply:
@yftacherzog Fixed.